### ISSUE TITLE
Add support for Nomad Variables

### ISSUE THREAD

**Issue by slonopotamus** [hashicorp/nomad-pack#409]
[Nomad Variables](https://developer.hashicorp.com/nomad/docs/concepts/variables)  is a lightweight alternative to Vault/Consul key-value storage with the added benefit of being scoped to Nomad namespaces.

**Comment by jrasell** [comment]
Closing in favour of #473

**Comment by slonopotamus** [comment]
No, this issue about a different thing. I want to be able to _create/update_ Nomad Variables via Nomad Pack. #473 wants to _read_ Nomad Variables when installing a pack.
